scripts/monitoring/cron-send-security-updates-count.py

```diff
@@ -81,8 +83,15 @@ def get_security_updates(self):
         if proc.returncode != 0:
             self.sec_updates = []
             raise subprocess.CalledProcessError(proc.returncode, subprocess.list2cmdline(cmd), output=stdout+stderr)
-        lineslist = [line for line in stdout.split('\n') if line and 'Errno' not in line]
-        self.sec_updates = [UpdateItem(*re.split(r'\s+', line)) for line in lineslist]
+        for line in stdout.split('\n'):
+            if line and 'Errno' not in line:
+                if len(re.split(r'\s+', line)) == 3:
+                    self.sec_updates.append(UpdateItem(*re.split(r'\s+', line)))
+                else:
```

Comment:
we're not supposed to exit non-zero, that hands errors to ops-runner which puts errors on our monitors that often get ignored for long periods (like the one this is fixing). In this case, we're logging that there's bad data, but that's not a fatal case, we can still pass data that we get later up to the monitor.